electrons/python/generic_electron_solver.py

```diff
@@ -0,0 +1,224 @@
+from __future__ import division
+from abc import ABC, abstractmethod
+from dataclasses import dataclass
+from numpy.typing import NDArray
+from typing import Tuple
+import numpy as np
+
+
+@dataclass
+class GenericElectronSolver(ABC):
+    # Simulation parameters
+    electron_density_per_cm3: float  # fluence-rate [/cm^2/s]
+    voltage_V: float  # [V/cm] magnitude of the electric field
```

Comment:
@jbrage should this be volts or volts/cm? The argument says that it should be volts, but the comment says otherwise.